cluster-operator/scripts/cluster_operator_run.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
 export JAVA_CLASSPATH=lib/io.strimzi.@project.build.finalName@.@project.packaging@:@project.dist.classpath@
 export JAVA_MAIN=io.strimzi.operator.cluster.Main
+
+if [ -f /opt/strimzi/custom-config/log4j2.properties ]; then
+    # if ConfigMap was not mounted and thus this file was not created, use properties file from the classpath
+    export JAVA_OPTS="${JAVA_OPTS} -Dlog4j2.configurationFile=file:/opt/strimzi/custom-config/log4j2.properties"
```

Comment:
Should we not instead always deploy the CM and pre-seed the CM with the properties from the properties file?
Easier to predict behaviour if we don't have this if check here, I can imagine us having to suggets users restart operator pod to make the operator run through this logic again.

Reply:
We are deploying it always. @scholzj thinks there may be some edge cases when the CM is not deployed/mounted as a volume. In that case, we should use at least some defaults (what is done by loading log4j2.properties from classpath).

Reply:
Can we `echo` something to say that a static logging configuration is being used because no config was found, and therefore that dynamic log config update won't work.